packages/zoe/src/contractFacet/seat.js

```diff
@@ -41,6 +53,14 @@ export const makeZcfSeatAdminKit = (
       assertExitedFalse();
       exited = true;
     },
+    setProposal: newProposal => {
```

Comment:
I prefer `init` rather than `set` for things that are in a distinct absent state before initialization (as opposed to having a default value) and cannot be mutated after being initialized. Our store abstraction follows this distinction between `init` and `set`. Our store abstraction also uses `get` consistently with `getProposal`. Getting before initialization throws.
```suggestion
    setProposal: initProposal => {
```

Reply:
I agree, the current API follows a pattern much more like init. I'll change this.